modules/understanding-node-cves-scanner-v4.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities/common-vuln-management-tasks.adoc
+
+:_mod-docs-content-type: CONCEPT
+// DO NOT RENAME THE ID - there are UI links using it in ACS
+[id="understanding-node-cves-scanner-v4_{context}"]
+= Understanding differences in scanning results between the Stackrox Scanner and Scanner V4
+
+== Summary
```

Comment:
I'm sorry, I missed this before - we aren't supposed to use headings like "Overview" or "Summary". I think we can actually just remove this because what follows is the introductory text - see the [mod docs guidance](https://redhat-documentation.github.io/modular-docs/#creating-concept-modules) (but to summarize - we don't need a heading here).